google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineStandardDeployTask.java

```diff
@@ -88,6 +89,10 @@ public void execute(ProcessStartListener startListener) {
           stagingDirectory,
           startListener,
           deploy(stagingDirectory, startListener));
+    } catch (AppEngineJavaComponentsNotInstalledException ex) {
+      deploy.getCallback().errorOccurred(
+          GctBundle.message("appengine.deployment.error.missing.java.components"));
+      logger.warn(ex);
```

Comment:
Confirmed - it does not generate an error report. Will merge when travis is done.